lms/djangoapps/branding/api.py

```diff
@@ -258,7 +278,7 @@ def _footer_navigation_links(language=settings.LANGUAGE_CODE):
             _(u"{platform_name} for Business").format(platform_name=platform_name)
         )),
         ("blog", (marketing_link("BLOG"), _("Blog"))),
-        ("help-center", (settings.SUPPORT_SITE_LINK, _("Help Center"))),
+        ("help-center", (_build_help_center_url(settings.LANGUAGE_CODE), _("Help Center"))),
```

Comment:
Same here, pass `language` instead of `settings.LANGUAGE_CODE`.

Reply:
Changed `settings.LANGUAGE_CODE` with `language`.